src/index.ts

```diff
@@ -61,11 +62,32 @@ class WalletProvider {
       getThemeConfig: () => {
         return themeConfig;
       },
+      sendPendingRequestCount: (count: Number) => {
```

Comment:
Prefer the primitive type `number` instead of the object type `Number`. The difference is only valuable in the cases when we create a number using the `Number` constructor `new Number(10)`.

```suggestion
      sendPendingRequestCount: (count: number) => {
```

Same change in the `onReceivingPendingRequestCount` method as well.

Reply:
See: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types.